modules/apps/object/object-test/src/testFunctional/tests/tests/PopulateObjectFormsConfiguration.testcase

```diff
@@ -23,41 +23,200 @@ definition {
+
+		PopulateObjectFormsConfiguration.selectCustomObject(label = "C_CustomObject");
+
+		Form.gotoAddField(fieldType = "Text");
+
+		PopulateObjectFormsConfiguration.gotoBuilderFormFields();
+
+		Form.gotoAddField(fieldType = "Date");
+
+		PopulateObjectFormsConfiguration.mapFormField(fieldName = "date");
+
+		PopulateObjectFormsConfiguration.publishFormObject();
+
+		PopulateObjectFormsConfiguration.viewUnmappedError(
+			fieldLabel = "Text",
+			fieldType = "Text");
+
+		PopulateObjectFormsConfiguration.closeUnmappedError();
+
+		PopulateObjectFormsConfiguration.saveFormObject();
 
+		PopulateObjectFormsConfiguration.viewUnmappedError(
+			fieldLabel = "Text",
+			fieldType = "Text");
 	}
 
 	@description = "LPS-133365 - Verify if is not possible to publish the Form when required Object field is not mapped to a Form Field in a Field Group"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CannotPublishWhenRequiredFieldNotMappedForFieldGroup {
 		property portal.acceptance = "true";
 
-		// TODO LPS-137353 CannotPublishWhenRequiredFieldNotMappedForFieldGroup pending implementation
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Required Field",
+			fieldName = "text",
+			fieldType = "String",
+			isRequired = "true",
+			objectName = "CustomObject");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Not Required Field",
+			fieldName = "date",
+			fieldType = "Date",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Not Required Field",
+			fieldName = "integer",
+			fieldType = "Integer",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		PopulateObjectFormsConfiguration.openForm();
 
+		PopulateObjectFormsConfiguration.openFormSettings();
+
+		PopulateObjectFormsConfiguration.selectObject(label = "Object");
+
+		PopulateObjectFormsConfiguration.selectCustomObject(label = "C_CustomObject");
+
+		Form.gotoAddField(fieldType = "Date");
+
+		PopulateObjectFormsConfiguration.mapFormField(fieldName = "date");
+
+		Form.closeSidebar();
+
+		Form.addFieldFromSidebarToFieldsGroup(
+			key_fieldLabel = "Date",
+			key_fieldName = "Numeric");
+
+		PopulateObjectFormsConfiguration.mapFormField(fieldName = "integer");
+
+		Form.closeSidebar();
+
+		Form.addFieldOutsideOfFieldsGroup(
+			fieldName = "Text",
+			index = "2");
+
+		PopulateObjectFormsConfiguration.publishFormObject();
+
+		PopulateObjectFormsConfiguration.viewUnmappedError(
+			fieldLabel = "Text",
+			fieldType = "Text");
+
+		PopulateObjectFormsConfiguration.closeUnmappedError();
+
+		PopulateObjectFormsConfiguration.saveFormObject();
+
+		PopulateObjectFormsConfiguration.viewUnmappedError(
+			fieldLabel = "Text",
+			fieldType = "Text");
```

Comment:
Update this with the macro that I created (viewUnmappedObjectRequiredFieldsError)